sites/healthcarepackaging.com/config/navigation.js

```diff
@@ -22,7 +22,7 @@ const mobileMenu = {
     { href: '/events', label: 'Events' },
     { href: '/downloads', label: 'Downloads' },
     { href: '/page/hcp-newsletters', label: 'Newsletters' },
-    { href: '/page/magazine', label: 'Magazine' },
+    { href: '/magazine', label: 'Magazine' },
```

Comment:
This is repeated here: https://github.com/parameter1/pmmi-media-group-websites/blob/d7e303ed653bc6f3de08c77a49592be8986d2ac7/sites/healthcarepackaging.com/config/navigation.js#L54 and should likely be changed there as well.